binance/base_client.py

```diff
@@ -206,6 +206,7 @@ def __init__(
     def _get_headers(self) -> Dict:
         headers = {
             "Accept": "application/json",
+            "Content-Type": "application/json",
```

Comment:
interesting at ccxt we're using `'Content-Type': 'application/x-www-form-urlencoded',`

Reply:
Interesting, yes, I would get it in test_options_price and test_options_mark_price

Reply:
according to the docs: 
```
For POST, PUT, and DELETE endpoints, the parameters may be sent as a query string or in the request body with content type application/x-www-form-urlencoded
```

i am wondering if we should remove this.. without it doesn't pass the tests.. but not sure if maybe with it it's failing and we just aren't seeing it in one of the skipped tests

Reply:
Just in case I edited the headers to be more similar to ccxt's implementation and what the docs say

Reply:
@pcriadoperez Should we also set it for get requests?